not-to-release/sources/answers/20111107092617AAgKm4X_ans.xml.conllu

```diff
@@ -710,7 +710,7 @@
 # text = The American people were NOT having any of it, and elected Richard Nixon to fix the mess LBJ had made (not to surrender like the antiwar movement was demanding).
 1	The	the	DET	DT	Definite=Def|PronType=Art	3	det	3:det	_
 2	American	American	ADJ	JJ	Degree=Pos	3	amod	3:amod	_
-3	people	people	NOUN	NNS	Number=Plur	6	nsubj	6:nsubj|12:nsubj	_
+3	people	person	NOUN	NNS	Number=Plur	6	nsubj	6:nsubj|12:nsubj	_
```

Comment:
Is this the nation sense of 'people'? You wouldn't say "the American individuals"

Reply:
"the American and Canadian peoples"?

Reply:
I don't have a ton of stake in this one - if we want certain ones to be "people" instead of "person", happy to make that change.  Is this the main one that sticks out to you, or was it more of a general change that needs to happen?